### PR TITLE
docs(risingwave): add backend docs page for RisingWave

### PR DESCRIPTION
**Pull request by gforsyth**
-------

## Description of changes

Adds a backend docs page for the RisingWave backend.

### CODE REVIEW

File: docs/backends/risingwave.qmd [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    database="database",
+)
+```
+
+::: {.callout-note}
+`ibis.risingwave.connect` is a thin wrapper around [`ibis.backends.risingwave.Backend.do_connect`](#ibis.backends.risingwave.Backend.do_connect).
+:::
+
+
+### Connection Parameters
```
Comment (lostmygithubaccount) [inline docs/backends/risingwave.qmd:88]: this should be "Connection parameters" per style guide (but seeing it's this for all the other backends, so maybe keep + fix this in a future PR)

File: docs/backends/risingwave.qmd [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    user="username",
+    password="password",
+    host="hostname",
+    port=4566,
+    database="database",
+)
+```
+
+::: {.callout-note}
+`ibis.risingwave.connect` is a thin wrapper around [`ibis.backends.risingwave.Backend.do_connect`](#ibis.backends.risingwave.Backend.do_connect).
```
Comment (lostmygithubaccount) [inline docs/backends/risingwave.qmd:84]: wrap line?

**Review by lostmygithubaccount — COMMENTED** [review]
minor things

File: docs/backends/risingwave.qmd (hunk 1 above)
Comment (gforsyth) [reply docs/backends/risingwave.qmd:89]: yeah, can handle that in a followup

**Review by cpcloud — APPROVED** [review]

**Pull request merged**